bottom-up-managed-apps/README.md

```diff
@@ -0,0 +1,53 @@
+**tl;dr Let’s offer only apps that we actually use as managed apps (of course, if customer also uses it or wants to use it), such as prometheus, oauth2-proxy, etc. and NOT: Linkerd, Harbor, etc. stuff on roadmap we don't actually use. This way, a small managed apps team can more effectively offer more managed apps for less effort, while retaining the same business and user value, which is reduce toil via managed apps. WDYT?**
+
+----
+
+This RFC arose from the idea to offer oauth2-proxy. It’s an app, which at least 2 of our customers  want (and I bet there are more), and I think we use. (See: [Slack](https://gigantic.slack.com/archives/CPC3M70UE/p1622729938024200))
+
+**Proposal: Offer managed apps, bottom-up, instead of top-down. Get back to our ‘original intention'.**
+
+Meaning:
+
+1. List the apps we already use
+2. Check which ones customers also already use or want to use
+3. Prioritize those as managed apps
+4. Grow our offering organically this way.
+
+This is in contrast to the apps in our roadmap. That is, observabilty → developer tooling -> use cases (Kafka, etc.)... most of which are apps we don't already use internally, or merely feel we ‘should’ use but actually don't.
+
+**Status Quo:**
+
+Assumption: The basis on which the managed apps team was created is this idea:
+
+> We use these apps already. Why not help customers with them as well? It won’t take much more effort.
```

Comment:
first apps might have started that way, that's also why we first wanted to prioritize prometheus (operator) as we had experience with prometheus already, but it was quite clear from the beginning that customers need more than what we need. That said starting with what we already have is a good idea as it's a quick win.